### DISCUSSION TITLE
Is there any way to delete users from Web interface?

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#3624] [Unsupported Versions] [answered]
Hello, I know you can disable user accounts, but is there any way to just delete the completely? In the so-common script, there seems to be a delete user function, but it is not implemented with "so-user".

**Comment by weslambert** [comment, marked as answer]
You should be able to do something like the following to delete a user:

`so-user delete user@needstobe.deleted`

**Reply by jertel** [reply]
This function is not documented since deleting users has the side effect of losing related data. For example, PCAP jobs that were created by a now non-existent user will show a blank area under the Owner column. Therefore if you don't want to lose historic audit data you should disable users instead of deleting them.

**Reply by thegreekman** [reply]
Hello, is there a command to delete a specific user from thehive? Something like: so-thehive-user delete?
Using so-user delete "username" does not work as I get an authentication error when it tries to remove the user from The Hive

**Comment by thegreekman** [comment]
The users I want to delete were test accounts I created, they have created
nothing. Is it ok to delete them now?

**Reply by jertel** [reply]
Based on your statement above it sounds like it's not a problem to delete them.

**Comment by thegreekman** [comment]
Ok, thanks.
